src/main/config/messages_template.xml

```diff
@@ -7,122 +7,127 @@ Copyright 2005, 2006 IBM Corporation
     <response></response>
   </message>
-  
-  <message id="DOTA069F" type="FATAL">
-    <reason>Input file '%1' cannot be located or read.</reason>
-    <response>Ensure that file was specified properly and that you have permission to access it.</response>
-  </message>
-  
+
   <message id="DOTA003F" type="FATAL">
     <reason>Cannot find the user specified XSLT stylesheet '%1'.</reason>
     <response></response>
   </message>
-  
+
   <message id="DOTA004F" type="FATAL">
     <reason>Invalid DITA topic extension '%1'.</reason>
     <response>Supported values are '.dita' and '.xml'.</response>
   </message>
-    
+
   <message id="DOTA011W" type="WARN">
     <reason>Argument "%1" is deprecated.</reason>
     <response>This argument is no longer supported in the toolkit.</response>
   </message>
+
   <message id="DOTA012W" type="WARN">
     <reason>Argument "%1" is deprecated.</reason>
     <response>Please use the argument "%2" instead.</response>
   </message>
+
   <message id="DOTA013F" type="FATAL">
     <reason>Cannot find the specified DITAVAL '%1'.</reason>
     <response></response>
   </message>
+
   <message id="DOTA014W" type="WARN">
     <reason>Attribute @%1 is deprecated.</reason>
     <response>Use attribute @%2 instead.</response>
   </message>
+
   <message id="DOTA015F" type="FATAL">
     <reason>Internal property %1 may not be set directly.</reason>
     <response>Use property %2 instead.</response>
   </message>
 
+  <message id="DOTA069F" type="FATAL">
+    <reason>Input file '%1' cannot be located or read.</reason>
+    <response>Ensure that file was specified properly and that you have permission to access it.</response>
+  </message>
+
   <!-- Start of Java Messages -->
-  
+
   <!-- DOTJ005F appears to indicate an internal programming error, failed to create
        a new module in ModuleFactory.java. This message should be updated
        to indicate it is an internal programming error and should be reported to the
-       toolkit development team. -->>
+       toolkit development team. -->
   <message id="DOTJ005F" type="FATAL">
     <reason>Failed to create new instance for '%1'.</reason>
     <response>Please ensure that '%1' exists and that you have permission to access it.</response>
   </message>
-  
-  <message id="DOTJ007I" type="INFO">
+
+  <message id="DOTJ007E" type="ERROR">
     <reason>Duplicate condition in filter file for rule '%1'.</reason>
     <response>The first encountered condition will be used.</response>
   </message>
-  
-  <message id="DOTJ007W" type="WARN">
+
+  <message id="DOTJ007I" type="INFO">
     <reason>Duplicate condition in filter file for rule '%1'.</reason>
     <response>The first encountered condition will be used.</response>
   </message>
 
-  <message id="DOTJ007E" type="ERROR">
+  <message id="DOTJ007W" type="WARN">
     <reason>Duplicate condition in filter file for rule '%1'.</reason>
     <response>The first encountered condition will be used.</response>
   </message>
-  
+
```

Comment:
@jelovirt @robander There are currently 3 identical instances of this message that differ only by type.

It looks like only `DOTJ007I` is actually referenced in the code. 

Can we safely remove `DOTJ007E` & `DOTJ007W`?

Reply:
Per earlier discussions, I think we would mark them as deprecated in 4.2, and then remove them after 4.2. It's possible (though unlikely with Java messages) that someone could have a custom plugin that still uses these message IDs... XSLT messages seem more likely to encounter that